ledger/participant-integration-api/src/main/resources/db/migration/h2database/V1__Append_only_schema.sql

```diff
@@ -399,12 +399,13 @@ CREATE TABLE participant_identity_provider_config
 -- Participant local store: users
 ---------------------------------------------------------------------------------------------------
 CREATE TABLE participant_users (
-    internal_id         INTEGER             GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    user_id             VARCHAR(256)        NOT NULL UNIQUE,
-    primary_party       VARCHAR(512),
-    is_deactivated      BOOLEAN             NOT NULL,
-    resource_version    BIGINT              NOT NULL,
-    created_at          BIGINT              NOT NULL
+    internal_id          INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    user_id              VARCHAR(256) NOT NULL UNIQUE,
+    primary_party        VARCHAR(512),
+    identity_provider_id VARCHAR(255) REFERENCES participant_identity_provider_config (identity_provider_id),
```

Comment:
Just in case, maybe have a test that tries to write the longest valid idp id?

Reply:
But it's impossible as `LedgerString` is concatenated on the SQL level and I cannot make `LedgerString` which is more than 255 chars :)

Reply:
I was thinking of a test trying to write an idp id of exactly max length, i.e. 255